openreview/venue/invitation.py

```diff
@@ -5052,4 +5052,127 @@ def set_edit_venue_group_invitations(self):
                     }
                 }
             )
-        )        
+        )
+
+    def set_venue_template_invitations(self):
+
+        super_id = self.venue.support_user.split('/')[0] 
+        template_domain = f'{super_id}/Template'
+        submission_deadline = tools.datetime_millis(self.venue.submission_stage.exp_date if self.venue.submission_stage.exp_date else datetime.datetime.now(datetime.timezone.utc)) 
+
+        self.client.post_invitation_edit(
+            invitations=f'{super_id}/-/Reviewers_Review_Count',
+            signatures=[template_domain],
+            content={
+                'venue_id': {'value': self.venue_id},
+                'reviewers_id': {'value': self.venue.get_reviewers_id() },
+                'activation_date': { 'value': submission_deadline + (60*60*1000*24*7*8) },
+            },
+            await_process=True
```

Comment:
Why do we need to wait for the process?

Reply:
the new configuration UI is creating invitations with process functions that create other invitations. I want to root process function to fail is some of their children fail. 

Here is an example:

1. deployment process
1.1. create submission invitation
1.1.1. create submission/dates invitation to edit the dates
1.2 create review invitation
1.2.1 create review/dates invitation to edit the dates
...